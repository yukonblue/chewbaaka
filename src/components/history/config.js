/**
 * history/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 07, 2020
 * Updated  : Aug 25, 2021
 */

export const config = {
  headMeta: {
    title: "History of the cheetah | Learn about cheetahs",
    description: "Learn about the history and evolution of the cheetah.",
    keywords: [
      "cheetah history",
      "cheetah evolution",
      "cheetah lineage",
      "cheetah population",
      "cheetah range",
    ]
  },
  pageProps: {
    coverImageNamePrefix: "History_Page_Cover_Image",
    title: "History",
    subtitle: "Learn about the evolution and migration of the species, its relationship with man throughout history, and its population and ranges.",
    pageMenuItems: [
      {
        title: "Evolution",
        tocImageFilename: "history_page_toc_menu_img_01-min.jpg"
      },
      {
        title: "Cheetah and Man",
        tocImageFilename: "history_page_toc_menu_img_02-min.jpg"
      },
      {
        title: "Range and Population",
        tocImageFilename: "history_page_toc_menu_img_03-min.jpg"
      }
    ],
    pageTailNavMenu: {
      prevPage: {
        label: "Home",
        href: "/"
      },
      nextPage: {
        label: "Biology",
        href: "/biology/"
      }
    }
  },
  contentPageIntro: {
    title: "History of the Cheetah",
    content: "The evolution and history of the cheetah are just as remarkable and " +
          "interesting as the species itself. Its evolution goes back to nearly " +
          "7 million years ago in time, and spanned across almost all the continents on the planet. " +
          "Its history with human interactions goes as far back as to 32000 BCE, and " +
          "the cheetah has been a symbol of significant meaning in different civilizations and " +
          "cultures across thousands of years of human history. Even up to this day, " +
          "the cheetah continues to be considered as a spiritual animal that manifests " +
          "as a symbol of a multitude of positive characteristics, such as adaptability, " +
          "persistence, and focus.",
    image: {
      filenamePrefix: "History_Page_Intro_Section_Background",
      aspectRatio: {
        width: 861,
        height: 420
      }
    }
  },
  contentPageSections: {
    section_Evolution: {
      intro: {
        title: "Evolution",
        content: "Although many would think the cheetah had originated on the continent of Africa, " +
        "the oldest fossils place  cheetahs in North America in what is now Texas, Nevada and Wyoming. " +
        "Cheetahs were common throughout Asia, Africa, Europe and North America until the end of the last " +
        "Ice Age, about 10,000 years ago, when massive climatic changes caused large numbers of mammals to disappear."
      },
      subsections: {
        subsection_EvolutionOfCats: {
          title: "Evolution of Cats"
        },
        subsection_CheetahEvolution: {
          title: "Cheetah Evolution"
        }
      },
    },
    section_CheetahAndMan: {
      intro: {
        title: "Cheetah and Man",
        content: "The history of cheetah-human relationship goes back in millennia, " +
        "and archeological artifacts uncovered throughout the world suggest that " +
        "human interactions with the species goes hand in hand with its migration and evolution. " +
        "The relationship between cheetahs and us have been both profound and complex, " +
        "as the species have been revered, utilized, displayed, and exploited throughout human history."
      },
      subsections: {
        subsection_RelationshipsWithMan: {
          title: "Relationships with Man",
          contents: {
            paragraph_Cheetah_And_Man_01:
              'Traditional African healers and witchdoctors used cheetah foot bones in spiritualistic rituals to symbolize fleet-footedness and speed. ' +
              'They used bones from a wide variety of animals and techniques varied from tribe to tribe. Objects ' +
              'represented a person, a thing or mood from the past, present or future, and are known as Divinatory sets.',
            paragraph_Cheetah_And_Man_02:
              'During the time of the Egyptian pharaohs, the cheetah was considered a goddess named ' +
              '"Mafdet". Pharaohs kept cheetahs as close companions, ' +
              "symbolic of Mafdet's protection of the royal throne.",
            paragraph_Cheetah_And_Man_03:
              'The San of Southern Africa ate cheetah meat as a symbol for speed, but it was not a main food in their diet.',
            paragraph_Cheetah_And_Man_04:
              'Kings wore cheetah skins for dignity. Trade in cheetah skin only started after European explorers began ' +
              'requesting them.',
          }
        },
        subsection_CheetahsInSports: {
          title: "Cheetahs In Sports",
          contents: {
            paragraph_The_Hunting_Leopard_01:
              'In c1700 BC, the Egyptians were the first to tame cheetahs. They admired the cheetah for its speed, ' +
              'hunting ability and beauty. They honored cheetahs as symbols of royalty and prestige.',

            paragraph_The_Hunting_Leopard_02:
              'The swiftest animal on earth became a cherished hunting companion of Pharaohs and royalty ' +
              'throughout Europe, Asia and India. Until the early 1900s, ownership of cheetahs was as important ' +
              'to these nobles as their love for gold. Cheetahs hunt by sight so they excelled in the sport ' +
              'known as "coursing". Hunts organized by royalty and nobles were for the challenge of sport, ' +
              'not for food. Hunts represented power and prestige.',

            paragraph_The_Hunting_Leopard_03:
              'By the 1500s the popularity of the cheetah as a hunting companion rivaled that of the dog. ' +
              'Cheetahs, the most easily tamed of the big cats, were caught, tamed and trained. Adults were ' +
              'used because cubs had not learned how to hunt. Tamed cheetahs formed a strong bond with their keepers.',

            paragraph_The_Hunting_Leopard_04:
              'Each cheetah rode to the hunt by horseback or on a cart. Its eyes were coved with a hood and uncovered ' +
              'when prey were sighted. The cat was released to chase down the prey then rewarded with meat fed from a ' +
              'wooden spoon.',

            paragraph_The_Hunting_Leopard_05:
              'Although cherished, pampered cheetahs were loved to near extinction and taken from the wild in great numbers. ' +
              'By the early 1900s, India and Iran were importing African cheetahs for the sport of "coursing", as their own ' +
              'wild population became too small.'
          }
        },
        subsection_CheetahsInArt: {
          title: "Cheetahs In Art",
          contents: {
            paragraph_Cheetah_In_Art_01:
              'Sumerians (Iraqi ancestors) were the first to use cheetahs in art.',

            paragraph_Cheetah_In_Art_02:
              'The cheetah and leopard have often been mistaken for each other. ' +
              'Cheetahs in early art were frequently called "panther" ' +
              'or "hunting leopard". Early artists called the true leopard "pardus" ' +
              'and described it as a cross between a cheetah (panther), and the lion (leo).',

            paragraph_Cheetah_In_Art_03:
              'Middle Age and Renaissance artists began drawing cheetahs and other ' +
              'animals more life-like. During this time of scientific exploration, ' +
              'detailed descriptions of animals provided incentive for reality in art. ' +
              'Artists included cheetahs more often in their paintings as the trade in ' +
              'live animals increased.',

            paragraph_Cheetah_In_Art_04:
              'Artists always pictured cheetahs as animals of speed and royalty.'
          }
        }
      }
    },
    section_PopulationAndRange: {
      intro: {
        title: "Range and Population",
        content: "A combination of factors such as game hunting, retaliatory killing, " +
                "habitat encroachment have caused the sharp decline of the global cheetah " +
                "population over the last century, and driven the species out of much of their " +
                "historical range. One of the last strongholds of the species today remain in " +
                "southern Africa, spanning over Namibia and Botswana, which together " +
                "holds two thirds of the species' population."
      },
      subsections: {
        subsection_Namibia: {
          title: "Namibia - Cheetah Capital of the World",
          contents: {
            Cheetah_Capital_Of_The_World: {
              title: "Namibia",
              subtitle: "Cheetah Capital of the World",
              content: {
                paragraph_Cheetah_Capital_Of_The_World_01:
                  "Namibia has the world's largest cheetah population. " +
                  "Approximately 3000 cheetahs share the land with humans, " +
                  "livestock, and wildlife.",

                paragraph_Cheetah_Capital_Of_The_World_02:
                  "Today, the status of the Namibian cheetah is stabilizing. " +
                  "During the 1980s, the population of Namibian cheetahs " +
                  "declined by half. In the 10-year period, nearly " +
                  "7,000 cheetahs were removed from the wild.",

                paragraph_Cheetah_Capital_Of_The_World_03:
                  "In most countries where cheetahs live, their numbers " +
                  "have been reduced to critical levels. The cheetah's " +
                  "survival worldwide is in human hands."
              }
            },
            Namibia_Home_For_The_Future: {
              title: "Namibia",
              subtitle: "Hope for the Future",
              content: {
                paragraph_01:
                  "The greatest hope for the cheetah's survival lies in the " +
                  'pristine countryside of Namibia. Nearly 1000 Namibian farmers ' +
                  "control the fate of the country's cheetahs and the land they live."
              }
            }
          }
        },
        subsection_RoadToExtinction: {
          title: "The Road to Extinction",
          contents: {
            The_Road_To_Extinction: {
              title: "The Road to Extinction?",
              subtitle: "1990",
              content: {
                paragraph_The_Road_To_Extinction_01:
                  "World development, industrialization, automobiles, and airplanes...",

                paragraph_The_Road_To_Extinction_02:
                  "Through the 1900s, man's inventions seemed limitless, yet there is " +
                  "nothing man-made that rivals the speed and efficiency of the cheetah.",

                paragraph_The_Road_To_Extinction_03:
                  "Scientists classified the cheetah as Acinonyx jubatus, yet nobles still " +
                  'referred to it as the "hunting leopard". By the end of the 1800s, cheetahs ' +
                  'were a rarity in Asia Minor and Arabia because of their use in the sport ' +
                  'of "coursing".',

                paragraph_The_Road_To_Extinction_04:
                  'Although it appeared that the cheetahs had a large range, their numbers ' +
                  'within that range existed in small pockets. Cheetahs, farmers and their livestock ' +
                  'all preferred open grasslands for their habitat. Increasing agricultural development ' +
                  'and new settlements played havoc with remaining cheetah populations.',
              }
            },
            Where_Did_The_Cheetah_Go: {
              title: "Where Did the Cheetah Go?",
              subtitle: "1975",
              content: {
                paragraph_Where_Did_The_Cheetah_Go_01:
                  'As human population grew, the amount of land devoted to livestock farming ' +
                  'steadily increased. Livestock filled the open land where cheetahs roamed. ' +
                  'Natural prey became scarce. Farmers kill other large predators. Although ' +
                  'game reserves protected them, cheetahs could not compete against hyenas ' +
                  'and lions. Farmlands offered cheetahs a safe haven, but they sometimes killed livestock.',

                paragraph_Where_Did_The_Cheetah_Go_02:
                  'Farmers saw cheetahs more frequently and thought their numbers had increased. ' +
                  'Cheetahs took the blame for most predator-related livestock losses. ' +
                  'Farmers killed cheetahs by the thousands as pests or to sell their skins ' +
                  'to the fur trade.',

                paragraph_Where_Did_The_Cheetah_Go_03:
                  'By 1975, researches realized that the cheetah was in trouble. CITES ' +
                  '(Convention on International Trade of Endangered Species) placed the cheetah ' +
                  'on Appendix I, making international trade in live cheetah or cheetah products illegal. ' +
                  'Local laws supported CITES in many countries where cheetahs still lived. ' +
                  'Researchers began looking for ways to encourage the growth of cheetah populations ' +
                  'through land management practices.'
              }
            }
          }
        }
      }
    }
  }
};
