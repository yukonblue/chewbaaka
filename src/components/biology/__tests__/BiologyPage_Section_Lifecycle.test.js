/**
 * BiologyPage_Section_Lifecycle.test.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 15, 2020
 * Updated  : Aug 20, 2021
 */

import React from 'react'

import { render } from '@testing-library/react'

import { config } from '../config'

import renderer from 'react-test-renderer'

import BiologyPageSectionLifecyle from '../BiologyPage_Section_Lifecycle'

test('renders BiologyPageSectionLifecyle component', () => {
  /**
   * Disable test because we currently use require.context
   * and it doesn't work in Jest.
   */
  // render(<BiologyPageSectionLifecyle config={config}/>);
});

test('BiologyPageSectionLifecyle component snapshot', () => {
  /**
   * Disable test because we currently use require.context
   * and it doesn't work in Jest.
   */
  // const tree = renderer
  //   .create(
  //     <BiologyPageSectionLifecyle config={config}/>)
  //   .toJSON();
  // expect(tree).toMatchSnapshot();
});
