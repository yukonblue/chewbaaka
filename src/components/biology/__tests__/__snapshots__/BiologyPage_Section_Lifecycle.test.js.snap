// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BiologyPageSectionLifecyle component snapshot 1`] = `
<article
  className="ContentPageSectionTemplateOuterContainer"
>
  <h2
    className="ContentPageSectionHeadTitle"
    id="lifecycle"
  >
    Lifecycle
  </h2>
  <p
    className="ContentPageHeadAndSectionIntroText"
  >
    One of the most incredible things about the cheetahs is the remarkable processes and transitions in their lifecycle. From born at an average weight of 350 grams and completely helpless and unable to see, to becoming one of the most skilled predators and the fastest land animal on the planet, nothing speaks more about the incredible adaptations that allow them to survive and thrive in the challenging environments where they live.
  </p>
  <div
    className="BiologyPageLifecycleDiagramOuterContainer"
  >
    <h3
      className="ContentPageSubsectionTitle TextCentered"
    >
      The Cycle of Life
    </h3>
    <p
      className="BiologyPageLifecycleDiagramSubtitle TextCentered"
    >
      There are four stages in the lifecycle of the cheetah.
    </p>
    <p
      className="BiologyPageLifecycleDiagramContentText TextCentered"
    >
      <b>
        Stage 1
      </b>
      : 90 to 95 days pregnancy; 
      <b>
        Stage 2
      </b>
      : 6 weeks to 18 months;
    </p>
    <p
      className="BiologyPageLifecycleDiagramContentText TextCentered"
    >
      <b>
        Stage 3
      </b>
      : 18 to 22 months; 
      <b>
        Stage 4
      </b>
      : adult life.
    </p>
    <img
      alt="lifecycle diagram"
      className="FullWidth DisplayBlock RetainAspectRatio Centered"
      height={675}
      src="Lifecycle_Diagram-min.png"
      width={640}
    />
  </div>
  <section
    className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
    style={
      Object {
        "backgroundColor": "#FFBB00",
      }
    }
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Stage 1
    </h3>
    <div
      className="ContentPageSideFloatFluidContainerOuterContainer"
    >
      <div
        className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
      >
        <div
          className="ImageViewOuterContainer"
          style={
            Object {
              "maxWidth": 320,
            }
          }
        >
          <div
            className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
          >
            <div
              className="ImageViewImgContainer"
            >
              <img
                alt="Cheetah cubs are born blind, so they need their mother's care."
                className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                height={480}
                src="cheetah_mom_carry_cub-min.jpg"
                width={320}
              />
            </div>
            <div
              className="ImageViewFamilyCaptionContainerStyles"
            >
              <div
                className="CaptionContainer"
                data-testid="CaptionComponentTestId"
                style={
                  Object {
                    "maxWidth": 320,
                  }
                }
              >
                <div
                  className="IconContainer"
                >
                  <i
                    aria-hidden="true"
                    className="image icon"
                    onClick={[Function]}
                  />
                </div>
                <p
                  className="ContentPageCaptionTextSize CaptionText"
                >
                  Cheetah cubs are born blind, so they need their mother's care.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSideFloatFluidContainerFixedPartContainer"
      >
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          The female’s gestation period lasts 90 to 95 days. This means she is pregnant for about three months.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Shortly before she is ready to give birth, the mother makes a den in a quiet hidden spot. She chooses her location in the tall grass and thick underbrush or near a clump of rocks.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          A cheetah gives birth to an average of five to six cubs. Each cub weights between 250 to 425 grams. Cubs are born completely helpless and with their eyes closed, but they develop rapidly. Scent and touch are used to find their mother’s nipples to suckle.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          They start crawling around the nest area at four to ten days when their eyes begin to open. At three weeks their teeth break through the gums.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          A mother cheetah will frequently move her litter. This prevents a build-up of scent of the den site, which may lead other predators to the cubs. She carries very young cubs in her jaws.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          The cubs are very vulnerable to lions, hyenas and other predators when the female leaves them alone. When hunting she may be away for up to 48 hours. In Tanzania’s Serengeti National Park, 90% of all cubs do not reach the age of 3 months! Other causes of death are abandonment when prey is scarce, exposure due to low temperatures or grass fires.
        </p>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
    style={
      Object {
        "backgroundColor": "#FFBB00",
      }
    }
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Stage 2
    </h3>
    <div>
      <div
        className="FluidTwoColumnContainerOuterContainer"
      >
        <div
          className="FluidTwoColumnContainerColumnContainer FluidTwoColumnLhsContainerColumnContainer"
        >
          <div
            className="ContentPageSubsectionPartContainer"
          >
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              At 1.5 to 2 months of age, the cubs leave the safety of the den to accompany their mother. They are very vulnerable as they are not able to defend themselves.
            </p>
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              They stop drinking their mother’s milk at 3 to 4 months of age. They start eating meat and learning to hunt. The games they play and the experiences they have during this stage will teach them skills needed to survive on their own.
            </p>
          </div>
        </div>
        <div
          className="FluidTwoColumnContainerColumnContainer FluidTwoColumnRhsContainerColumnContainer"
        >
          <div
            className="ContentPageSubsectionPartContainer"
          >
            <div>
              <a
                href="https://www.youtube.com/embed/Wjtb7XMZlgY"
              >
                <button
                  className="ui blue icon button"
                  onClick={[Function]}
                >
                  Cheetah Mom Teaches Cubs to Hunt 
                  <i
                    aria-hidden="true"
                    className="youtube icon"
                    onClick={[Function]}
                  />
                </button>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
    style={
      Object {
        "backgroundColor": "#FFBB00",
      }
    }
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Learning to Hunt
    </h3>
    <div
      className="ContentPageSideFloatFluidContainerOuterContainer"
    >
      <div
        className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
      >
        <div
          className="ImageViewOuterContainer"
          style={
            Object {
              "maxWidth": 480,
            }
          }
        >
          <div
            className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
          >
            <div
              className="ImageViewImgContainer"
            >
              <img
                alt="Cheetah cubs learning how to hunt."
                className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                height={320}
                src="cheetah_cubs_learning_hunting-min.jpg"
                width={480}
              />
            </div>
            <div
              className="ImageViewFamilyCaptionContainerStyles"
            >
              <div
                className="CaptionContainer"
                data-testid="CaptionComponentTestId"
                style={
                  Object {
                    "maxWidth": 480,
                  }
                }
              >
                <div
                  className="IconContainer"
                >
                  <i
                    aria-hidden="true"
                    className="image icon"
                    onClick={[Function]}
                  />
                </div>
                <p
                  className="ContentPageCaptionTextSize CaptionText"
                >
                  Cheetah cubs learning how to hunt.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSideFloatFluidContainerFixedPartContainer FloatLeft"
      >
        <div
          className="TextBubbleOuterContainerFlexWrapper"
        >
          <div
            className="TextBubbleOuterContainer"
            style={
              Object {
                "backgroundColor": "rgb(undefined)",
                "maxWidth": 520,
                "minHeight": 520,
              }
            }
          >
            <div
              className="TextBubbleInnerContainer"
            >
              <div
                className="TextBubbleInnerCore"
              >
                <h1
                  style={
                    Object {
                      "color": "rgb(undefined)",
                      "fontSize": 41.6,
                    }
                  }
                >
                  Learning to Hunt
                </h1>
                <p
                  style={
                    Object {
                      "color": "rgb(undefined)",
                      "fontSize": 21,
                    }
                  }
                >
                  The mother brings live prey, such as a young gazelle, to the 9 to 12 month old cubs. She releases it in front of them and the cubs attempt to catch it. 
                  This allows the cubs to practice their hunting skills while still under her supervision. Accurate timing and coordination during a hunt are important for their future survival. 
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
    style={
      Object {
        "backgroundColor": "#FFBB00",
      }
    }
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Stage 3
    </h3>
    <div>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        Stage three begins at the age 18 to 22 months when the cubs have grown to sub-adults and leave their mother. The sub-adults will remain together for up to six more months. At first, their success rate at capturing prey is poor.
      </p>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        Cheetahs are diurnal, hunting mornings and early evenings. They rely on their sight to find prey. They spend most of the day resting under shady trees or on termite mounds. Night hunting is only done during a bright moon.
      </p>
      <div>
        <a
          href="https://www.nationalgeographic.co.uk/animals/2017/12/stunning-pictures-cheetahs-action"
        >
          <button
            className="ui blue icon button"
            onClick={[Function]}
          >
            Stunning Pictures of Cheetahs in Action | National Geographic
            <i
              aria-hidden="true"
              className="file image icon"
              onClick={[Function]}
            />
          </button>
        </a>
      </div>
      <img
        alt="What is diurnal?"
        className="FactBannerImage FullWidth DisplayBlock VerticalCushionPadding Centered FactBannerSmallDimension"
        height={480}
        src="What_is_Diurnal_L-min.png"
        width={960}
      />
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
    style={
      Object {
        "backgroundColor": "#FFBB00",
      }
    }
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Stage 4
    </h3>
    <div>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        In Stage 4, cheetahs become sexually mature. Although they are mature at 16 to 18 months, most do not breed until they are three to five years old.
      </p>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        At 20 - 30 months of age, females leave their litter-mates to find suitable mates and start their own families. They raise their families on their own without the help of the males.
      </p>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        Males usually do not breed until they are 4 to 5 years of age, and dominant in a territory. They live alone, or brothers form permanent groups called "coalitions". These groups stay together for life, claim territories, hunt and find mates together.
      </p>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
    style={
      Object {
        "backgroundColor": "#FFBB00",
      }
    }
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Finding a Mate
    </h3>
    <div
      className="ContentPageSideFloatFluidContainerOuterContainer"
    >
      <div
        className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
      >
        <div
          className="ImageViewOuterContainer"
          style={
            Object {
              "maxWidth": 720,
            }
          }
        >
          <div
            className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
          >
            <div
              className="ImageViewImgContainer"
            >
              <img
                alt="Cheetahs often mark their scent on tree trunks with their urine. This behavior is common among cats."
                className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                height={480}
                src="cheetah_scent_marking-min.jpg"
                width={720}
              />
            </div>
            <div
              className="ImageViewFamilyCaptionContainerStyles"
            >
              <div
                className="CaptionContainer"
                data-testid="CaptionComponentTestId"
                style={
                  Object {
                    "maxWidth": 720,
                  }
                }
              >
                <div
                  className="IconContainer"
                >
                  <i
                    aria-hidden="true"
                    className="image icon"
                    onClick={[Function]}
                  />
                </div>
                <p
                  className="ContentPageCaptionTextSize CaptionText"
                >
                  Cheetahs often mark their scent on tree trunks with their urine. This behavior is common among cats.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSideFloatFluidContainerFixedPartContainer"
      >
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          The range of a female offspring may partially overlap that of her mother. Namibian cheetahs are more social than those reported in other countries. Females are often seen with multiple adults and cubs of varying ages.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Female cheetahs are polyoestrous, which means there is no regular breeding season. If not bred, females come into heat (estrus) several times a year. Estrus means they are ready to breed. If cubs are lost to predators females soon come into estrus again.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Smell, sound, and behavioral stimuli attract males to females. Female cheetahs leave a scent trail by releasing sex hormones in urine and feces. They mark trees and bushes. This behavior increases during courtship.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          When courtship takes place, males will follow females closely and mock fighting may be observed.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          When a female is ready to mate, she adopts a receptive posture. The male mounts the female, bites the back of her neck, and breeding takes place. When the male dismounts the female she rolls over on her back and swats at him.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Mating will take place for one to several days and ends when the male loses interest in the female and leaves. Males do not help raise the cubs.
        </p>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
    style={
      Object {
        "backgroundColor": "#FFBB00",
      }
    }
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Living Fast - Dying Young
    </h3>
    <div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div
          className="ContentPageSubsectionTwoColumnContentTemplateOuterContainer"
        >
          <div
            className="ContentPageSubsectionTwoColumnContentTemplateInnerContainer"
          >
            <div
              className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
            >
              <div>
                <h4
                  className="ContentPageSubsectionSubtitle"
                >
                  Male coalitions
                </h4>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  A coalition consists of a group of male siblings of the same litter or young unrelated males that have joined together. A hierarchy develops among the males within a coalition.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Young males are usually chased away from their birth range by dominant breeding males. They may establish home ranges more than 100 km away. A coalition is more successful in acquiring and holding territories and in defending kills than single males. This competition can result in mortality among males.
                </p>
              </div>
            </div>
            <div
              className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
            >
              <div>
                <div
                  className="ImageViewOuterContainer"
                  style={
                    Object {
                      "maxWidth": 510,
                    }
                  }
                >
                  <div
                    className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
                  >
                    <div
                      className="ImageViewImgContainer"
                    >
                      <img
                        alt="A coalition of cheetahs on the lookout."
                        className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                        height={342}
                        src="cheetah_coalition-min.jpg"
                        width={510}
                      />
                    </div>
                    <div
                      className="ImageViewFamilyCaptionContainerStyles"
                    >
                      <div
                        className="CaptionContainer"
                        data-testid="CaptionComponentTestId"
                        style={
                          Object {
                            "maxWidth": 510,
                          }
                        }
                      >
                        <div
                          className="IconContainer"
                        >
                          <i
                            aria-hidden="true"
                            className="image icon"
                            onClick={[Function]}
                          />
                        </div>
                        <p
                          className="ContentPageCaptionTextSize CaptionText"
                        >
                          A coalition of cheetahs on the lookout.
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div
          className="ContentPageSubsectionTwoColumnContentTemplateOuterContainer"
        >
          <div
            className="ContentPageSubsectionTwoColumnContentTemplateInnerContainer"
          >
            <div
              className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
            >
              <div>
                <div
                  className="ImageViewOuterContainer"
                  style={
                    Object {
                      "maxWidth": 640,
                    }
                  }
                >
                  <div
                    className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
                  >
                    <div
                      className="ImageViewImgContainer"
                    >
                      <img
                        alt="Cheetahs often have to come to face hyenas in competition of food."
                        className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                        height={424}
                        src="cheetah_face_hyena-min.jpg"
                        width={640}
                      />
                    </div>
                    <div
                      className="ImageViewFamilyCaptionContainerStyles"
                    >
                      <div
                        className="CaptionContainer"
                        data-testid="CaptionComponentTestId"
                        style={
                          Object {
                            "maxWidth": 640,
                          }
                        }
                      >
                        <div
                          className="IconContainer"
                        >
                          <i
                            aria-hidden="true"
                            className="image icon"
                            onClick={[Function]}
                          />
                        </div>
                        <p
                          className="ContentPageCaptionTextSize CaptionText"
                        >
                          Cheetahs often have to come to face hyenas in competition of food. (Image credit: Steve Volkwyn)
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer"
            >
              <div>
                <h4
                  className="ContentPageSubsectionSubtitle"
                >
                  Mortality
                </h4>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  The cheetah’s lifespan is poorly documented in the wild. A radio collared cheetah lived to be almost 7 years of age in Tanzania’s Serengeti National Park, and one of CCF’s radio collared cheetahs in Namibia lived for over 10 years.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Adult mortality is one of the most significant limiting factors for cheetah population growth and survival. Poaching, competition with large predators and farmers, and loss of habitats and prey are factors attributing to early death.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Though cub deaths are high, cheetahs have evolved to reproduce rapidly in response to this mortality.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</article>
`;
